static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx

```diff
@@ -33,7 +33,12 @@ type IssueProps = {
 
 const MAX_DISPLAYED_ISSUES_COUNT = 10;
 
-const MIN_ISSUES_TABLE_WIDTH = 600;
+const TABLE_WIDTH_BREAKPOINTS = {
+  FIRST: 700,
+  SECOND: 600,
```

Comment:
Lets be more aggressive here. Judging by the screencap, it feels like it should happen even sooner